### DISCUSSION TITLE
W/Firestore(25709): (24.4.0) [WriteStream]: (3513ff8) Stream closed with status: Status{code=INTERNAL, description=error in frame handler, cause=java.lang.NoSuchMethodError:

### DISCUSSION THREAD

**Discussion by Sara-IM** [firebase/flutterfire#10337] [Q&A]
I am using firebase firestore, it was working correctly before, but this problem has appeared and i dont know how to solve it
![image](https://user-images.githubusercontent.com/93736072/215163837-625a8784-d623-4adb-94cf-f1d619e8e30f.png)

**Comment by Luisca67** [comment]
Lograste resolverlo?
